presto-hive/src/test/java/com/facebook/presto/hive/TestPrestoS3FileSystem.java

```diff
@@ -311,6 +312,13 @@ public void testEncryptionMaterialsProvider()
         }
     }
 
+    @Test(expectedExceptions = UnrecoverableS3OperationException.class, expectedExceptionsMessageRegExp = ".*(Path: /tmp/test/path).*")
```

Comment:
This regex should really be
```java
".*\\Q (Path: /tmp/test/path)\\E"
```
We need to escape the parenthesis, since they are part of the expected message, not the regex.